eng/mgmt/mgmtmetadata/resources_resource-manager.txt

```diff
@@ -3,12 +3,12 @@ AutoRest installed successfully.
 Commencing code generation
 Generating CSharp code
 Executing AutoRest command
-cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/resources/resource-manager/readme.md --csharp --version=v2 --reflect-api-versions --tag=package-policy-2021-06 --csharp.output-folder=E:\github-repos\calec-azure-sdk-for-net\sdk\resources\Microsoft.Azure.Management.Resource\src\GeneratedPolicy202106
```

Comment:
I saw in the [review comment](https://github.com/Azure/azure-sdk-for-net/pull/22508#discussion_r695310613), we should not change to `Feature`. And that change is supposed to be reverted. Can you double check the change here? We changed to use `-tag-package-features-2021-07`.

Reply:
this is auto generated whenever I run ./generate.ps1. So it is by design.